src/pages/Download.js

```diff
@@ -1,9 +1,238 @@
-import { SectionTitle } from '../common/Section'
+import {
+  GridSection,
+  GridSectionTitle,
+  Section,
+  SectionTitle,
+  WideHSection,
+} from '../common/Section'
+
+import { BulletList, BulletListInside } from '../common/List'
+import Button from '../common/Button'
+
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs'
+import 'react-tabs/style/react-tabs.css'
+
+import { isWinPhone, isIOS, isWindows, isMacOs } from 'react-device-detect'
+import { useState } from 'react'
+
+import logo_name from '../assets/images/logo_web_name.png'
 
 const Download = () => {
+  var sta
```

Comment:
It's always better to use `let` instead of `var` in modern JS (IntelliJ also complains about it).

And it doesn't need to be run on every rerender, so maybe we can pull the block outside the component to make it clear (althought it wouldn't actually impact performance).

Reply:
Done the variable part

Reply:
block goes outside